.github/workflows/release.yml

```diff
@@ -0,0 +1,26 @@
+name: Releases
+
+on: 
+  push:
+    tags:
+    - '*'
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
```

Comment:
Should probably be `actions/checkout@v2`

Reply:
Copy and paste from the action's page, but good one.